docs/pages/academy/1-GRAKN-intro/1-grakn-intro.md

```diff
@@ -61,9 +61,9 @@ In this section you will find questions and exercises that you should be able to
 
 For this lesson, the exercises are quite free-form, so feel free to answer them as you see fit:
 
-  * What are the main components of a knowledge based systems?
+  * What are the main components of a knowledge graphd systems?
```

Comment:
:goat:

Reply:
Fixed in https://github.com/graknlabs/grakn/pull/2681/commits/8532f9fbb7d8c3bd3d12d1239d37759be0efea0f